README.md

```diff
@@ -1,9 +1,9 @@
-# central-object-store
+# object-store-lib
 
 [![Git Commit](https://img.shields.io/github/last-commit/mojaloop/central-object-store.svg?style=flat)](https://github.com/mojaloop/central-object-store/commits/master)
```

Comment:
@lewisdaly what about the names here? should we rename them later?

Reply:
Oops! I forgot those in my search and replace.